README.md

```diff
@@ -109,6 +109,6 @@ s<-algo_10(dfm,val,"users_id")
 
 
 ```
-image_read("C:/Users/fredo/Documents/Code Projet/Flowchart_DLA.JPG")
-
+x<-image_read("C:/Users/fredo/Documents/Code Projet/Flowchart_DLA.JPG")
```

Comment:
Tu travailles encore en path absolu ici :sweat_smile:
Lis sur les path absolu et les path relatifs (absolute paths and relative paths)
En soit tu veux utiliser uniquement des path relatifs car les path absolu ne foncitonnent que sur ton pc. Donc tes liens ne peuvent que pointer vers des élements qui sont dans le dossier du projet dla.
```C:/Users/fredo/Documents/Code Projet``` :point_left: ça c'est uniquement sur ton pc

Reply:
@Louisf-Redon :point_up: